src/components/Navigation/components/Item/tests/Item.test.tsx

```diff
@@ -358,6 +358,55 @@ describe('<Nav.Item />', () => {
         .exists(),
     ).toBe(true);
   });
+
+  describe('small screens', () => {
+    let matchMedia: jest.SpyInstance;
+    beforeEach(() => {
+      matchMedia = jest.spyOn(window, 'matchMedia');
```

Comment:
matchMedia is a window function used to match media queries in javascript. Item uses it in -> https://github.com/Shopify/polaris-react/blob/3ab258a9d064de9c6ad9c75e805b1e4f6f7241e3/src/components/Navigation/components/Item/Item.tsx#L301 to determine screen size